src/main/java/org/example/daos/EmployeeDao.java

```diff
@@ -0,0 +1,73 @@
+package org.example.daos;
+
+import org.example.models.Employee;
+import org.example.models.EmployeeRequest;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.PreparedStatement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeDao {
+
+    public List<Employee> getAllEmployees() throws SQLException {
+        List<Employee> employees = new ArrayList<>();
+
+        try (Connection connection = DatabaseConnector.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(
+                    "select employeeID, employeeName ,employeeSalary, "
+                            + "employeeBankAccountNumber,"
+                            + "employeeNationalInsuranceNumber,"
+                            + "employeeCommissionRate from Employees;");
+
+            while (resultSet.next()) {
+                Employee order = new Employee(
+                        resultSet.getInt(
+                                "employeeID"),
+                        resultSet.getString(
+                                "employeeName"),
+                        resultSet.getDouble(
+                                "employeeSalary"),
+                        resultSet.getString(
+                                "employeeBankAccountNumber"),
+                        resultSet.getString(
+                                "employeeNationalInsuranceNumber"));
+
+                employees.add(order);
+            }
+        }
+        return employees;
+    }
+
+    public int createEmployee(final EmployeeRequest employeeRequest)
+            throws SQLException {
+
+        Connection connection = DatabaseConnector.getConnection();
```

Comment:
Some error handling needs added here and lets close the connection in a finally block

Reply:
put the connectio in a try block, which will close it. thx